beta/project-griffon/set-up-project-griffon.md

```diff
@@ -44,8 +63,34 @@ import ACPGriffonBeta // <-- import the Project Griffon library
 ### Register Project Griffon with Mobile Core
 
 {% tabs %}
+{% tab title="Android" %}
+Registering the extension with Core sends Experience Platform SDK events to an active Project Griffon session. To start using the extension library, you must register the extension with the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension.
+
+#### Java
+
+1. Register the extension wherever you are registering other extensions
+```java
+AndroidGriffonBridge.registerExtension();
+```
+
+2. Call Griffon setup with the application reference.  Do this as early as possible in your application startup.
```

Comment:
please reword so that it reads as follows:

1. Call the Griffon set up with the application reference.

  **Tip**: Complete this step as early as possible after your application starts up.

Reply:
Done.